demisto_sdk/commands/lint/linter.py

```diff
@@ -181,6 +190,23 @@ def __init__(
                     "Unable to find yml file in package"
                 )
 
+    def should_disable_network(self):
+        pack_name = get_pack_name(str(self._pack_abs_dir))
+        _pack_ignore_file_path = Path(
+            f"Packs/{pack_name}/{PACKS_PACK_IGNORE_FILE_NAME}"
+        )
+        if _pack_ignore_file_path.exists():
+            config = ConfigParser(allow_no_value=True)
+            config.read(_pack_ignore_file_path)
+            if TESTS_REQUIRE_NETWORK_PACK_IGNORE in config.sections():
+                ignored_integrations_scripts_ids = config[
+                    TESTS_REQUIRE_NETWORK_PACK_IGNORE
+                ]
+                _id = self._yml_file_content.get("commonfields", {}).get("id")
+                if _id in ignored_integrations_scripts_ids:
```

Comment:
please use `get_id` here instead of getting it manually

Reply:
if you mean a function, there isn't such one, exists only in `demisto_sdk/commands/common/content/objects/abstract_objects/dictionary_based_object.py`

but its a method that is bounded to an object, in my case its a raw yml file.

Reply:
@ilaner maybe we can use the graph for that?

Reply:
why would we need to use the graph here?

Reply:
never mind - but please extract the logic there into an external function (unbound it from the object), possibly move it to `tools.py` and re-use it here. One day I'll make it to implement https://github.com/demisto/demisto-sdk/pull/2345, but until then.. let's not make _more_ duplicates of this logic.

Reply:
so to get things straight, would you like me to extract `self._yml_file_content.get("commonfields", {}).get("id")` to an external function?

Reply:
extract the one at `dictionary_based_object.py` into `tools.py`, and use it both here and there.

Reply:
done